pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -152,6 +152,10 @@ func validateRule(rule *admissionregistration.Rule, fldPath *field.Path, allowSu
 	return allErrors
 }
 
+// AcceptedAdmissionReviewVersions contains the list of AdmissionReview versions the *prior* version of the API server understands.
```

Comment:
this is to make sure we can safely downgrade apiserver one version? I was surprised that you cannot create a configuration that only talks v1 in 1.16

Reply:
to be able to safely downgrade, and to ensure a skewed HA cluster of 1.15 and 1.16 API servers does not permit creation of webhooks that will break half the API servers in the cluster